templates/telco/osm/onboarding-network-functions.html

```diff
@@ -188,11 +188,9 @@ <h3 class="p-heading--4">OSM in KubeCon 2021, LA</h3>
 <section class="p-strip--suru-accent is-dark">
   <div class="row">
     <div class="col-12">
-      <div class="p-heading-icon--small">
-        <h3 class="p-heading--4" id="tell-us">Tell us about your network function</h3>
-        <p>Canonical can help you automate your network function onboarding to Charmed OSM. Please choose the relevant options for your network functions to see how it can work for you.</p>
-        <p><a href="https://charmed-osm.com/#tell-us" class="p-button--positive">Start now</a></p>
-      </div>
+      <h3 class="p-heading--4" id="tell-us">Tell us about your network function</h3>
+      <p>Canonical can help you automate your network function onboarding to Charmed OSM. Please choose the relevant options for your network functions to see how it can work for you.</p>
+      <p><a href="https://charmed-osm.com/#tell-us" class="p-button--positive">Start now</a></p>
```

Comment:
This link should be: https://charmed-osm.com/contact-us

Reply:
```suggestion
      <p><a href="https://charmed-osm.com/contact-us" class="p-button--positive">Start now</a></p>
```